### PR TITLE
[Console] improved code coverage of Command class

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

This PR increases the code coverage of the Command class from ``86.82%`` to ``96.90%`` of 2.7 branch.

After also https://github.com/symfony/symfony/pull/21067 was merged, thecode coverage of the Command class has been improved from 86.32% to 96.61% of master branch.

before:
before

after:
after

### CODE REVIEW

File: src/Symfony/Component/Console/Tests/Command/CommandTest.php [hunk 1]
```diff
@@ -156,6 +164,13 @@ public function testGetSetAliases()
         $this->assertEquals(array('name1'), $command->getAliases(), '->setAliases() sets the aliases');
     }
 
+    public function testSetAliasesNull()
+    {
+        $command = new \TestCommand();
+        $this->expectException(\InvalidArgumentException::class);
```
Comment (Simperfit) [inline src/Symfony/Component/Console/Tests/Command/CommandTest.php]: This is not possible in PHP 5.3 you should use the FQCN.

**Review by Simperfit — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Console/Tests/Command/CommandTest.php (hunk 1 above)
Comment (stof) [inline src/Symfony/Component/Console/Tests/Command/CommandTest.php]: Well, ``InvalidArgumentException``

Comment (ShinDarth) [inline src/Symfony/Component/Console/Tests/Command/CommandTest.php]: any clue?

**Comment by ShinDarth** [comment]
@stof all good now, I guess it's ready to be merged.

**Review by linaori — APPROVED** [review]

**Pull request closed without merging**